### DISCUSSION TITLE
Add ChromaDB vector store driver support

### DISCUSSION THREAD

**Discussion by KamilFatkhiev** [griptape-ai/griptape#1533] [Ideas]
- [x] I have read and agree to the [contributing guidelines](https://github.com/griptape-ai/griptape#contributing).

Hi!
I'm very interested in Griptape for its ease of building multi-agent applications, however I haven't found a ChromaDB driver in the current version of the framework. I think it would be good to add support for this vector storage.

**Comment by collindutter** [comment]
Hey @KamilFatkhiev, thanks for the suggestion -- this seems like it would be a great addition to Griptape. While I can't provide a great timeline right now, if you're interested in implementing bits yourself the [PineconeVectorStoreDriver](https://github.com/griptape-ai/griptape/blob/f6ab3ed627b44e443f55b8125297c2011d3b5ae0/griptape/drivers/vector/pinecone_vector_store_driver.py#L16) should serve as a great reference point.

Please don't hesitate to reach out if you have any questions!

**Comment by LuciAkirami** [comment]
Can I try working on this @collindutter ? I have some experience integrating vector stores in some open-source AI libraries.

**Comment by collindutter** [comment]
Hey @LuciAkirami, go for it! Feel free to reach out on Discord if you have any questions. Thanks!
